### ISSUE TITLE
system: hardware information scrollbars

### ISSUE THREAD

**Issue by dcrdev** [cockpit-project/cockpit#8577]
There's a slight display defect with the latest hardware information feature under system. With chrome 64.0.3282.140 and a display width of 1920 - going to this page, navigating away and then coming back to the hardware information page, the vertical scrollbar will no longer display i.e.

1) Login to cockpit
2) Navigate to hardware information (scrollbars show)
3) Go to another menu item - let's say "Logs" for instance.
4) Go back to hardware information (no scrollbars) - scrolling still works using click wheel/page up/down.

If I reduce the width of the window - there's no issue with scrollbars displaying.

**Comment by garrett** [comment]
I was able to reproduce this in the same version of Chrome (64.0.3282.140, the current stable version), regardless of window size.

While flipping back and forth (with the browser navigation arrows, page back and forward), I noticed the same bug once happened in the log view as well, so I don't think this is specific to the hardware information view.

Each time, I could get scrollbars to show up again by resizing the browser.

Perhaps it's some JavaScript layout code that's triggered on initial page load and resize, but not triggered when back and forward are used?
